src/explore-education-statistics-admin/src/pages/publication/PublicationDetailsPage.tsx

```diff
@@ -60,7 +60,7 @@ const PublicationDetailsPage = () => {
           <SummaryList>
             <SummaryListItem term="Publication title">{title}</SummaryListItem>
             <SummaryListItem term="Publication summary">
-              {summary}
+              {summary ?? 'No summary set'}
```

Comment:
Could use `Not set` as the default value here, given that it's already labelled as the summary field.